mobile-center-crashes/Crashes.js

```diff
@@ -53,9 +56,11 @@ let Crashes = {
         return RNCrashes.setEnabled(shouldEnable);
     },
 
-    notifyWithUserConfirmation(userConfirmation) {
+    notifyUserConfirmation(userConfirmation) {
         RNCrashes.notifyWithUserConfirmation(userConfirmation);
-        Helper.sendErrorAttachments(getErrorAttachmentsMethod, filteredReports);
+        if (userConfirmation !== UserConfirmation.DontSend) {
```

Comment:
Because `userConfirmation` is external user input and it can be any value or `undefined`, can we validate this value just to be safe?

Reply:
@dhei please create stories then as native SDKs don't check and they are the ones they should check.